### PR TITLE
fix: CORE-1549 - Cannot run canvases in dev environment:

### PR DESCRIPTION
**Pull request by mikhaelsantos**
* Fix data-api configuration
* Add twitter-bitcoin

Both Historical and Realtime canvases following the tutorial are fixed in dev environment.

### CODE REVIEW

File: streamr-docker-dev/bin.sh [hunk 1]
```diff
@@ -22,7 +22,7 @@ HELP=0
 
 SERVICE_COMBINATIONS=(
 ""
-"cassandra kafka init_keyspace mysql redis smtp zookeeper nginx broker data-api"
+"cassandra kafka init_keyspace mysql redis smtp zookeeper nginx broker"
```
Comment (Aapzu) [inline streamr-docker-dev/bin.sh]: Why this change? In the docs we recommend to use this option (option `1`) when developing `engine-and-editor`, and `data-api` is needed for that

Comment (mikhaelsantos) [reply streamr-docker-dev/bin.sh]: Are you talking about backend or frontend development?

Comment (Aapzu) [reply streamr-docker-dev/bin.sh]: Backend

Comment (Aapzu) [reply streamr-docker-dev/bin.sh]: Hmm, I think that it occurs mostly in frontend development, true

Comment (mikhaelsantos) [reply streamr-docker-dev/bin.sh]: After consulting with Eric it can affect the backend development.
